elements/header.js

```diff
@@ -103,7 +103,7 @@ HeaderElement.prototype.createElement = function (props) {
   return html`
       <header class="${header}">
         <div class="fr relative">
-          ${importButton.render({
+          ${importButton.createElement({
```

Comment:
this should still use `importButton.render`, `createElement` is a private api. See https://github.com/choojs/nanocomponent#usage